e/egeria/egeria_ubuntu_20.04.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : egeria
+# Version       : egeria-release-2.10 
+# Source repo   : https://github.com/odpi/egeria
+# Tested on     : Ubuntu 20.04.1 LTS (Focal Fossa)
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+export URL=https://github.com/odpi/egeria.git
+export BRANCH="$1"
+PKG_NAME=${URL##*/}
+PKG_NAME=${PKG_NAME%%.*}
+
+if [ -d $PKG_NAME ] ; then
+  rm -rf $PKG_NAME
+fi
+# Verify the arguments passed from command line.
+
+if [ $# -lt 1 ]
+then
+  echo "Usage: $0 branch or tag_name >> For specific versions, default is master"
+  BRANCH="master"
+fi
+#echo "Branch is $1 and no of arguments $# "
+if [ $# -gt 1 ]
+then
+        echo "Specify no of arguments for specific versions by default it will be master, USAGE: $0 BRANCH"
```

Comment:
Pl. rephrase

Reply:
@gururajrkatti Please let me know what shall I update it with.

Reply:
Specify only one argument for building the specific version ??